src/cmd_line.c

```diff
@@ -49,6 +51,11 @@ void cmd_line_attempt(const char *line) {
         cmd_line_ls(the_rest);
     } else if (strcmp("cat", first_word) == 0) {
         cmd_line_cat(the_rest);
+    } else if (strcmp("process", first_word) == 0) { // temporary, for debugging
+        struct process_permissions test_permissions;
+        test_permissions.max_number_of_pages = 10;
```

Comment:
I think we need to have people mallocing the permissions struct right? Since the permissions need to persist for the entire life of the process, we can't rely on this object always being in scope. 

I feel like we should follow that convention here so we don't lead people down the wrong path